.github/workflows/functional-test.yaml

```diff
@@ -198,7 +198,7 @@ jobs:
             * Dapr: ${{ env.DAPR_VER }}
             * Azure KeyVault CSI driver: ${{ env.AZURE_KEYVAULT_CSI_DRIVER_VER }}
             * Azure Workload identity webhook: ${{ env.AZURE_WORKLOAD_IDENTITY_WEBHOOK_VER }}
-            * Bicep recipe location `${{ env.BICEP_RECIPE_REGISTRY }}/test/functional/shared/recipes/<name>:${{ env.REL_VERSION }}`
+            * Bicep recipe location `${{ env.BICEP_RECIPE_REGISTRY }}/test/functional-portable/corerp/recipes/<name>:${{ env.REL_VERSION }}`
```

Comment:
Question about this - how do the bicep recipes work for tests outside of the `corerp` bucket?

It's confusing if all the recipes we use in tests have `corerp` in their path. Can you explain how this should work?